README.md

```diff
@@ -9,60 +9,31 @@
 [![AppVeyor](https://ci.appveyor.com/api/projects/status/github/googleapis/nodejs-vision?branch=master&svg=true)](https://ci.appveyor.com/project/googleapis/nodejs-vision)
```

Comment:
Where are these changes to the root README coming from?

I just checked out the `master` of `nodejs-vision` and ran `npm run generate-scaffolding` in the root & there were no updates to the root README

Reply:
Sorry I'm not sure where that came from. Reverting in next commit.

Reply:
Fixed in latest commit.